### PR TITLE
Bug vpn client config

### PR DESCRIPTION
**Pull request by gogondi1**
## Overview/Summary
unable to configure vpn_client_configuration due to revoked_certificate block defined with wrong parameters #791

in hub and spoke model when try to configure vpn_client_configuration under virtual_network_gateway, error is receiving due to one of the parameter under revoked_certificate is wrong, public_cert_data under revoked_certificate block need to replace by "thumbprint" according to terraform documentation. 

[Screenshot 2023-08-16 at 14 55 12(url)
## This PR fixes/adds/changes/removes

1.  you can configure vpn_client_config successfully 

### Breaking Changes

1. if issue with current modification replace thumbprint with public_cert_data  

## Testing Evidence

Please provide any testing evidence to show that your Pull Request works/fixes as described and planned (include screenshots, if appropriate).

[Screenshot 2023-08-17 at 15 36 42(url)

[Screenshot 2023-08-17 at 15 34 16(url)
## As part of this Pull Request I have

- [x ] Checked for duplicate [Pull Requests](https://github.com/Azure/terraform-azurerm-caf-enterprise-scale/pulls)
- [ x] Associated it with relevant [issues](https://github.com/Azure/terraform-azurerm-caf-enterprise-scale/issues), for tracking and closure.
- [x ] Ensured my code/branch is up-to-date with the latest changes in the `main` [branch](https://github.com/Azure/terraform-azurerm-caf-enterprise-scale/tree/main)
- [x ] Performed testing and provided evidence.
- [x ] Updated relevant and associated documentation.

### CODE REVIEW

**Comment by gogondi1** [comment]
@microsoft-github-policy-service agree

File: .github/policies/cla.yml
Comment (jtracey93) [inline .github/policies/cla.yml:1]: Can this file be removed please?

File: .github/policies/platformcontext.yml
Comment (jtracey93) [inline .github/policies/platformcontext.yml:1]: can this file be removed please?

**Review by jtracey93 — CHANGES_REQUESTED** [review]

File: .github/policies/platformcontext.yml
Comment (gogondi1) [reply .github/policies/platformcontext.yml:1]: yes

**Review by gogondi1 — COMMENTED** [review]
I have delete cla.yml and platformcontext.yml files, could anyone please review

**Review by gogondi1 — COMMENTED** [review]
I request to review the changes and merge the branches

**Review by matt-FFFFFF — APPROVED** [review]

**Comment by matt-FFFFFF** [comment]
Thank you for this pr - since I could not push to your fork to update from main, I pulled your changes into a new branch that I can work on. Therefore closing in favour of #835

**Pull request closed without merging**